wrapper.ts

```diff
@@ -70,27 +79,29 @@ export class Pty {
     // Transfer ownership of the FD to us.
     this.#fd = this.#pty.takeFd();
 
-    this.#socket = new ReadStream(this.#fd);
-    const userFacingRead = new PassThrough();
-    const userFacingWrite = new PassThrough();
-    this.#socket.pipe(userFacingRead);
-    userFacingWrite.pipe(this.#socket);
-    this.read = userFacingRead;
-    this.write = userFacingWrite;
+    this.#socket = new ReadStream(this.#fd)
```

Comment:
```suggestion
    this.#socket = new ReadStream(this.#fd);
```

ewwww semicolonless code.